### PR TITLE
fang lizhi reads landau in 1970

### PR DESCRIPTION
**Pull request by LenaHenningsen**
> Pull-requests should be ready to merge, that is free of conflicts, and they should pass our testsuite. If you require a specific reviewer you can request a review on the right.

### Please indicate issues related to this PR (if any)
See # .
Close # .

### Checklist
-   The title line explains what these changes do
-   All my changes are marked in the `last_modfied` column
-   The PR has no merge conflicts
-   Travis checks are passing

### CODE REVIEW

**Comment by LenaHenningsen** [comment]
Tried to fix the things. Have to go no

File: data/Person.csv [hunk 1]
```diff
@@ -1230,3 +1230,6 @@ P0805,康塔尔斯基,,zh,male,NULL,NULL,NULL,NULL,1931,2017,NULL,NULL,PL0443,NU
 P0806,Kontarsky,Alfons,de,male,NULL,NULL,NULL,NULL,1932,2010,NULL,NULL,PL0443,NULL,NULL,https://en.wikipedia.org/wiki/Aloys_and_Alfons_Kontarsky,,,,,,2019-02-18,MG,2019-02-18,MG
 P0806,康塔尔斯基,,zh,male,NULL,NULL,NULL,NULL,1932,2010,NULL,NULL,PL0443,NULL,NULL,https://en.wikipedia.org/wiki/Aloys_and_Alfons_Kontarsky,,,,,,2019-02-18,MG,2019-02-18,MG
 P0807,Li,Yizhe,en,male,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,NULL,SS00255,170-173,,,"Li Yizhe refers to a group of three authors: Li Zhengtian, ChenYiyang, Wang Xizhe, Guo Hongzhi",,2019-02-18,LH,2019-02-18,LH
+P0808,Fang,Lizhi,en,male,NULL,NULL,NULL,NULL,1936,2012,NULL,NULL,PL0002,NULL,NULL,SS00261,,,,Fang Lizhi was not rusticated but sent for reeducation,,2019-02-19,LH,2019-02-19,LH
+P0808,方,励之,zh,male,NULL,NULL,NULL,NULL,1936,2012,NULL,NULL,PL0002,NULL,NULL,SS00261,,,,Fang Lizhi was not rusticated but sent for reeducation,,2019-02-19,LH,2019-02-19,LH
+P0809,Landau,Lev,en,male,NULL,NULL,NULL,NULL,1908,1968,NULL,NULL,PL0444,NULL,NULL,https://en.wikipedia.org/wiki/Lev_Landau,,,,,,2019-02-19,LH,2019-02-19,LH
```
Comment (duncdrum) [inline data/Person.csv]:
```suggestion
P0809,Ландау,Лев Давидович,ru,male,NULL,NULL,NULL,NULL,1908,1968,NULL,NULL,PL0444,NULL,NULL,https://en.wikipedia.org/wiki/Lev_Landau,,,,,,2019-02-19,LH,2019-02-19,LH
```

**Comment by LenaHenningsen** [comment]
Thanks!

**Comment by duncdrum** [comment]
no more auto-format 💃

**Pull request merged**